app/src/pages/inside/common/accordion/accordion.jsx

```diff
@@ -31,25 +31,21 @@ export const Accordion = ({ tabs, headerClassNames, contentClassNames, toggleTab
             key={tab.id}
             tab={tab}
             onClick={() => toggleTab(tab.id)}
-            headerClassNames={cx(headerClassNames, {
+            headerClassNames={cx({
```

Comment:
Since we no longer need `headerClassNames` and `contentClassNames`, there is no need to pass them as props to the child component. We should work with `classNames` directly in `AccordionTab` component using property from `tab` props.